### PR TITLE
Task 386-387 : Semeval2018 task3

### PR DESCRIPTION
**Pull request by amirrezamirzaei**
task 386 & 387 using [Semeval2018 task3](https://aclanthology.org/S18-1005.pdf)
task 387 is a multi class(3) classification task but the labels aren't balanced, So Im not very sure about it

### CODE REVIEW

**Comment by danyaljj** [comment]
@XudongOliverShen wondering if you can help with this PR's review.

File: tasks/task387_semeval_2018_task3_irony_classification.json [hunk 1]
```diff
@@ -0,0 +1,11486 @@
...
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "SemEval 2018 task3 (\"https://aclanthology.org/S18-1005.pdf\")"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task you are given a tweet that contains some form of irony. You must classify the type of irony the tweet has. Label the tweets (\"polarity\",\"situational\",\"other\") based on the irony they have.Situational irony happens when a situation fails to meet some expectations, Label these instances as \"situational\". polarity irony happens when irony is achieved by inverting the intended sentence, Label these instances as \"polarity\". there are other kind of ironies which are neither polarity or situational, Label these instances as \"other\". ",
```
Comment (XudongOliverShen) [inline tasks/task387_semeval_2018_task3_irony_classification.json]: missing space before "Situational"

**Review by XudongOliverShen — COMMENTED** [review]
For task 387, maybe can be made balanced according to the class with the smallest number of instances, e.g., sampling equal number of instances from other classes. That's the way I did when I created other tasks. : )

other than this, looks good to me!

**Comment by amirrezamirzaei** [comment]
doing that will reduce the instances a lot. currently, the instances in each class are:
polarity : 1389 instance
situational : 315 instance
other : 204 instance
reducing the polarity instances to 500 should be oky I think.

**Comment by XudongOliverShen** [comment]
500 + 315 + 204 instances sounds good to me!

**Comment by danyaljj** [comment]
Thank you @amirrezamirzaei  for the PR and @XudongOliverShen for the review!

**Pull request merged**